webextensions/api/dom.json

```diff
@@ -0,0 +1,28 @@
+{
+  "webextensions": {
+    "api": {
+      "dom": {
+        "openOrClosedShadowRoot": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/dom/openOrClosedShadowRoot",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "63"
```

Comment:
Firefox does not support the `dom` API namespace.

But it does support the `element.openOrClosedShadowRoot` property.

Reply:
How do you suppose we should handle this?  Notes?  Alt. names?

Reply:
Is `alternative_name` typically used for same API signatures? The difference here is a method vs a getter property.

How about `partial_implementation` with the note that it's available as `element.isOpenOrClosedShadowRoot` instead of a method in the extension namespace?

If customary, we could even use alternative_name + partial_implementation + notes.

Reply:
It's hard to classify what is partial implementation, since that property refers to "an incomplete implementation of the spec that causes significant browser compatibility issues"...but since web extensions have no spec to refer to...!

However, if this method has a completely different structure between Firefox and Chrome, I'm actually thinking that we should split it into two separate features.